src/Loader.js

```diff
@@ -53,11 +53,20 @@ export async function load(
   let modelData
   const urlStr = url.toString()
   if (urlStr.startsWith('file://') && process.env.APP_ENV !== 'prod') {
-    const path = urlStr.substring('file://'.length)
-    const stats = fs.statSync(path)
-    const fileSize = stats.size
-    modelData = fs.readFileSync(path, {flag: 'r'})
-  } else {
+    // Decode the URL to handle any percent-encoded characters
+    const decodedPath = decodeURIComponent(urlStr.substring('file://'.length));
```

Comment:
I'd pull this out into a helper to make it easy to test path -> path, without invoking the file open